.travis.yml

```diff
@@ -9,7 +9,9 @@ addons:
 before_install:
   - docker build -t nginx .
   - docker run -d -u 1234:0 --net=host -p 8081:8081 --name nginx -e NGINX_HEADER_Some-Header='some "value"'
-       -e NGINX_ALWAYS_HEADER_Some__Always__Header='some always "value"' nginx
+       -e NGINX_ALWAYS_HEADER_Some__Always__Header='some always "value"'
+       -e IP_WHITELIST='10.0.0.3 10.5.8.0/24 22.88.88.1-22.88.88.7'
```

Comment:
what IPs are those?

Reply:
This are just arbitrary IPs used by the integration tests.